graphx/src/main/scala/org/apache/spark/graphx/lib/PageRank.scala

```diff
@@ -89,6 +114,10 @@ object PageRank extends Logging {
       // Set the vertex attributes to the initial pagerank values
       .mapVertices( (id, attr) => resetProb )
 
+    val personalized = srcId isDefined
+    val src:VertexId = srcId.getOrElse(-1L)
+    def delta(u:VertexId, v:VertexId):Double = { if (u == v) 1.0 else 0.0 }
```

Comment:
Minor style comment: there should be a space after the colon here and on the line above.